explainer.md

```diff
@@ -160,23 +160,27 @@ await navigator.queryHandwritingRecognizerSupport('supportedTypes')
 ### Perform Recognition
 
 ```JavaScript
-// Optional hints to the recognizer.
-const optionalHints = {
+// Model constraints determine the handwriting recognition model
+// used to create the recognizer.
+const modelConstraints = {
```

Comment:
Maybe "modelSpecification" is a bit better name?

Reply:
Let's use constraint for now. Specification would probably should lead to a determined / universally agreed model being chosen.

In the handwriting case, the exact model would probably differ across platforms.